atlp-rwanda/space-bn-backend
# Ft signin

### What does this PR do?
- Implement user signin and generate a token.
### Description of Task to be completed?
- Implement the signin.
- Implement the JWT token generation
### How should this be manually tested?
- Clone the repository.
- In your home directory  move  into the cloned folder by **cd ft-signin**
- Then run **npm install** to install all dependencies.
- Run **npm start** to start the server.
- access the swagger documentation on: **localhost:5000/api-docs**
-  after successful login u can use a middleware "check-auth.js" to protect routes with JWT
- Run **npm run test** to run the test.
### Any background context you want to provide?
- N/A
### What are the relevant Pivotal Tracker stories?
- [#175743315](https://www.pivotaltracker.com/story/show/175743315)
### Screenshots (if appropriate)
- N/A
### Questions
- N/A
